server/api/impl/proxy/asset.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package proxy
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/api/impl/aws"
+)
+
+func (p *Proxy) GetAsset(appID apps.AppID, assetName string) (*http.Response, error) {
```

Comment:
It's a bit odd forge a `http.Response` here, given that we only want to communicate a SatusCode and a Body. Should return `io.ReadCloser, int, error` instead?

Reply:
Sure, but I did not want to loose the headers from the http apps.

Reply:
Then my questions boils down to if we need the headers or not: https://github.com/mattermost/mattermost-plugin-apps/pull/53#discussion_r583874562